postgres-appliance/scripts/configure_spilo.py

```diff
@@ -314,6 +314,13 @@ def deep_update(a, b):
     command: /scripts/basebackup.sh
     retries: 2
 {{/USE_WALE}}
+{{#STANDBY_CLUSTER}}
+  create_replica_method:
```

Comment:
You are right, if `USE_WALE` is not set, there will be no `postgresql.basebackup_fast_xlog`.

But, we don't want to have some duplicates in the config if both, `USE_WALE` and `STANDBY_CLUSTER` are set.